docs/pinot_hadoop.rst

```diff
@@ -79,13 +79,13 @@ workflow to generate Pinot segments.
 .. code-block:: none
 
   mvn clean install -DskipTests -Pbuild-shaded-jar
-  hadoop jar pinot-hadoop-0.016-shaded.jar SegmentCreation job.properties
+  hadoop jar pinot-hadoop-0.1.0-shaded.jar SegmentCreation job.properties
 
 You can then use the SegmentTarPush job to push segments via the controller REST API.
 
 .. code-block:: none
 
-  hadoop jar pinot-hadoop-0.016-shaded.jar SegmentTarPush job.properties
+  hadoop jar pinot-hadoop-0.1.0-SNAPSHOT-shaded.jar SegmentTarPush job.properties
```

Comment:
Should the shaded jar names be the same, either `pinot-hadoop-0.1.0-shaded.jar` or `pinot-hadoop-0.1.0-SNAPSHOT-shaded.jar`?

Reply:
instead of `0.1.0`, we can put `<version>` since this will going to be changed periodically

Reply:
when i generated, it became snapshot

Reply:
ill put version here but leave it for the other one since the other docs are based on this version